File: week1/BOJ_11399/ATM_홍지훈.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

const input = [];

rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  const N = Number(input[0]);
  const P = input[1].split(" ").map(Number);

  P.sort((a, b) => a - b);

  console.log(P.reduce((acc, val, idx) => (acc += val * (N - idx)), 0));

  process.exit();
});

File: week1/BOJ_11399/ATM_홍지훈.py
N = int(input()) # 사람의 수 1 ~ 1000
P = [int(x) for x in input().split()] # 인출하는데 걸리는 시간 1 ~ 1000

# i 번째 사람이 돈을 인출하는데 걸리는 시간 Pi

P.sort()
sum = 0

for i in range(0, N):
  sum += P[i] * ( N - i )

print(sum)